docs/buildRTD.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
`super nitpicky` : can we rename filename to `build_rtd.sh` for consistency (and avoid camelcase)?

Reply:
sure, will update the BUILD_DIR and conf.py accordingly